src/openapi/spec/unleash-context-schema.ts

```diff
@@ -11,7 +11,7 @@ export const schema = {
         properties: {
             type: 'object',
             additionalProperties: {
-                anyOf: [{ type: 'string' }, { type: 'number' }],
+                type: 'string',
```

Comment:
haven't we lost the array here?

Reply:
that's the purpose. For the SDK it does not matter if the type is string or number. It will accept both anyway, and convert if needed. It is also a mismatch with the post-schema only requiring type "string" for additional context properties.